core/src/main/java/io/fabric8/maven/core/util/kubernetes/KindFilenameMapperUtil.java

```diff
@@ -27,10 +32,20 @@
     public static Map<String, List<String>> loadMappings() {
 
         final String location = "/META-INF/fabric8/kind-filename-type-mapping-default.adoc";
+        final String locationMappingProperties =
+            EnvUtil.getEnvVarOrSystemProperty("fabric8.mapping", "/META-INF/fabric8/kind-filename-type-mapping-default.properties");
 
-        try (final InputStream mappingFile = loadContent(location)) {
+        try (final InputStream mappingFile = loadContent(location); final InputStream mappingPropertiesFile = loadContent(locationMappingProperties)) {
```

Comment:
just a question: do you know how try-catch for resources deal with null elements. I mean, is this then ignore or does it result in an NPE when the jvm tries to close() it ?

Reply:
Ignores.